setup.py

```diff
@@ -97,4 +97,6 @@ def package_tree(pkgroot):
               op.join('io', 'artemis123', 'resources', '*.csv'),
               op.join('io', 'edf', 'gdf_encodes.txt')
           ]},
-          scripts=['bin/mne'])
+          entry_points={'console_scripts': [
+              'mne = mne.commands.utils:main',
+          ]})
```

Comment:
@agramfort I deleted `bin/mne` and added this, it seems to be the modern way to specify command entry points. Azure is happy and I can confirm locally on Windows `mne` does not work on 0.19 and on this PR with this change it does. But just wanted to point this out so you could look and not be surprised when `bin` is gone :)

Reply:
nice !